nemo-cli/src/cli.rs

```diff
@@ -103,6 +103,12 @@ pub(crate) struct OutputArgs {
     /// does not affect export directives that already specify a compression
     #[arg(short, long = "gzip", default_value = "false")]
     gz: bool,
+    /// Specify which predicates to dump
+    #[arg(long = "dump", value_enum)]
+    pub(crate) dumping: Option<Dumping>,
```

Comment:
I think I'd rather give this a name that clarifies that this is dumping facts.
For example, `dump-facts` might do it. Maybe even `print-facts` since this leads to writing the facts to stdout.
The internal naming should be changed accordingly but is not as critical as the user-facing argument names.